assets/pages/hello-world/helloWorld.js

```diff
@@ -0,0 +1,11 @@
+
+"use strict";
+
+import React  from "react";
```

Comment:
Are we using double or single quotes? Should the lint rules enforce consistency?